CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# [v0.6.0]
+
+## Changed
+- Crate is now using edition 2021.
+
+## Added
+- Derived `Eq` for `Error` types and the `PvhBootCapability` enum.
+
+## Fixed
+- Fixed a bug in `load_cmdline` due to which the command line was not null
```

Comment:
We should probably mention about [this bug](https://github.com/rust-vmm/linux-loader/pull/118#discussion_r976388631) as well.

Reply:
We should really enforce separate commits for different bug fixes, otherwise it's very hard to not miss them in the changelog. Can you help me with formulating a changelog entry? I am still not 100% sure I understood what the problem was there.

Reply:
Fixed an off-by-one error in load_cmdline, where we were doing
validations on the first address after the command line memory region,
instead of the last inclusive one of it.

Reply:
Thanks @lauralt. I've added it, please have another look.